zuul.d/projects.yaml

```diff
@@ -1005,6 +1004,13 @@
         - workshops-tox-integration-smart-mgmt
         - workshops-tox-integration-windows
 
+- project:
+    name: github.com/openshift/community.okd
+    default-branch: main
+    templates:
+      - ansible-collections-community-okd
+      - publish-to-galaxy-3pci
```

Comment:
so, these templates will need to be updated to use the 3pci-check pipeline

Reply:
I believe these will still use check

Reply:
@pabelanger what will happen if we stay with check?

Reply:
I don't expect them to work properly, given we have different github app permissions. I'd like to keep them in 3pci-check too, given check / gate usually work together. We also have different priorities setup for pipelines.

Reply:
to add on more here, because I'm tired.  We wouldn't even trigger on check jobs for new PRs on the project, because we only have https://github.com/ansible/project-config/blob/master/zuul.d/pipelines.yaml#L345 configured on 3pci-check

Reply:
Is there an example somewhere of a template that uses the 3pci-check pipeline? I'm feeling pretty lost here and not sure what needs to be done.

Reply:
@gravesm our jobs are associated with the `check` pipeline, https://zuul-ci.org/docs/zuul/reference/pipeline_def.html

The `ansible-collections-community-okd` already uses the right pipeline for historical reason:
https://github.com/ansible/ansible-zuul-jobs/blob/master/zuul.d/project-templates.yaml#L429